### PR TITLE
Enable websock on some servers; Use the typical branches for Sepolia

### PR DESCRIPTION
**Pull request by zah**

### CODE REVIEW

**Comment by zah** [comment]
Requires https://github.com/status-im/nimbus-eth2/pull/3820 to be merged in nimbus-eth2's `unstable` branch.

**Comment by Menduist** [comment]
Let me fix https://github.com/status-im/nimbus-eth2/issues/3814 first, if possible
(will be done this week)

File: ansible/group_vars/nimbus.prater.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ beacon_node_repo_branch: >-
        ((node.branch == "libp2p") | ternary("nim-libp2p-auto-bump-unstable", node.branch))
      ) }}
 beacon_node_nim_commit: '{{ node.get("nim_commit", "") }}'
+beacon_node_build_nim_flags: '-d:noSignalHandler {{ node.get("nim_flags", "") }}'
```
Comment (jakubgs) [inline ansible/group_vars/nimbus.prater.yml:10]: I the `-d:testnet_servers_image` no longer necessary?

**Review by jakubgs — APPROVED** [review]

File: ansible/group_vars/nimbus.prater.yml (hunk 1 above)
Comment (jakubgs) [reply ansible/group_vars/nimbus.prater.yml:10]: We do add it by default to builds:
https://github.com/status-im/infra-role-beacon-node-linux/blob/323a957d14891447843ba76fee06aae9d120fa55/defaults/main.yml#L25

Comment (zah) [reply ansible/group_vars/nimbus.prater.yml:10]: Yes, it doesn't have any effect at the moment.

**Comment by zah** [comment]
The planned fix by @Menduist has been merged, so this is ready for applying

**Pull request merged**